docs/guide/front-end-assets.md

```diff
@@ -68,18 +73,55 @@ The output will **sing** before it **dances**. If the modules are instantiated i
 
 the output will **dance** before it **sings**.
 
+**"Why do I have to export a function?"** A funny thing about JavaScript `import` statements: they don't guarantee any order in which the files are loaded. To fix that, Apostrophe requires you to export a function from each `ui/src/index.js` file so that it can call them in the order the modules are initialized.
+
+## Ordering the Sass output files
+
+The same rules apply to Sass files. Rules found in or imported by `ui/src/index.scss` files are compiled in the order the modules are activated in `app.js`. Of course, your Sass files should just contain CSS rules. Sass imports always execute in the order given, so you don't have to worry about exporting anything.
+
 ::: note The Boilerplate
-The [Apostrophe 3 boilerplate](https://github.com/apostrophecms/a3-boilerplate/) follows our recommended strategy:
-1. use webpack to build your assets
-2. push the end result to A3's asset pipeline
+The [Apostrophe 3 boilerplate](https://github.com/apostrophecms/a3-boilerplate/) takes advantage of `ui/src` in exactly the same way. The `asset` module contains `ui/src/index.js` and `ui/src/index.scss` files, which Apostrophe automatically discovers.
+:::
+
+## Supporting Internet Explorer 11
+
+Internet Explorer 11 is [going away soon](https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/#:~:text=With%20Microsoft%20Edge%20capable%20of,certain%20versions%20of%20Windows%2010.), but for now at least some projects still require support for it.
 
-**Here is how development works using our boilerplate project**:
+By default, Apostrophe does *not* guarantee that code written in modern JavaScript in `ui/src/index.js` files will run in Internet Explorer 11. However, you can easily turn on support by setting the `es5: true` option of the `@apostrophecms/asset` module:
```

Comment:
> in `ui/src/index.js` files

Could we call it "the default build process" or something like that? Something easier to remember than what a file path means.

Reply:
No "easily." Once someone starts struggling to find it they feel dumber since it was supposed to be easy.